mycore-base/src/main/java/org/mycore/frontend/cli/MCRDeveloperCommands.java

```diff
@@ -149,4 +153,37 @@ public static void showResource(String uri) {
         }
     }
 
+
+    @MCRCommand(
+            syntax = "touch object {0}",
+            help = "Touch object with id {0}",
+            order = 80
+    )
+    public static void touchObject(String id) {
+        try {
+            MCRObjectID objectId = MCRObjectID.getInstance(id);
+            MCRObject object = MCRMetadataManager.retrieveMCRObject(objectId);
+            MCRMetadataManager.update(object);
```

Comment:
the repair command calls here:
MCRMetadataManager.fireRepairEvent(obj);

So in difference to our repair commands not only all event handlers are fired again. The additional code handling the structure section is also running in this case.